config/management/operational/src/test_helper.rs

```diff
@@ -6,7 +6,10 @@ use libra_config::config;
 use libra_crypto::{ed25519::Ed25519PublicKey, x25519};
 use libra_management::{error::Error, secure_backend::DISK, TransactionContext};
 use libra_network_address::NetworkAddress;
-use libra_types::{account_address::AccountAddress, chain_id::ChainId};
+use libra_types::{
```

Comment:
we need to make this closer to storage_helper in the genesis tooling...

Reply:
Okay. Will leave this for a follow up PR.